<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gebruikersinfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gebruikersinformatie</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gebruiker-ID (volledig): </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPlainTextEdit" name="useridLabel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="crcLabel">
        <property name="text">
         <string>Gebruiker-ID (kort): </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Gebruikersnaam: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <widget class="QPushButton" name="friendButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>zet op vriendenlijst</string>
      </property>
      <property name="icon">
       <iconset resource="res/icons.qrc">
        <normaloff>:/icons/vriend.png</normaloff>:/icons/vriend.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="banButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>bannen</string>
      </property>
      <property name="icon">
       <iconset resource="res/icons.qrc">
        <normaloff>:/icons/ban.png</normaloff>:/icons/ban.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAan_vrienden_toevoegen">
   <property name="text">
    <string>Aan vrienden toevoegen</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
